commerce-service/src/main/java/com/liferay/commerce/model/impl/CommerceOrderItemImpl.java

```diff
@@ -69,32 +70,40 @@ public CProduct getCProduct() throws PortalException {
 	public CommerceMoney getDiscountAmountMoney() throws PortalException {
 		CommerceOrder commerceOrder = getCommerceOrder();
 
+		CommerceCurrency commerceCurrency = commerceOrder.getCommerceCurrency();
```

Comment:
@luca-pellizzon We have to refactor Commerce Money and commerce Price to use the code as well